Metadata/AmazonMetadataBuilder.php

```diff
@@ -27,12 +36,12 @@ class AmazonMetadataBuilder implements MetadataBuilderInterface
      * @var string[]
      */
     protected $acl = array(
-        'private' => CannedAcl::PRIVATE_ACCESS,
-        'public' => CannedAcl::PUBLIC_READ,
-        'open' => CannedAcl::PUBLIC_READ_WRITE,
-        'auth_read' => CannedAcl::AUTHENTICATED_READ,
-        'owner_read' => CannedAcl::BUCKET_OWNER_READ,
-        'owner_full_control' => CannedAcl::BUCKET_OWNER_FULL_CONTROL,
+        'private' => self::PRIVATE_ACCESS,
```

Comment:
using static would result in `static:: is not allowed in compile-time constants`